web/modules/custom/bglobal_pardot/templates/bglobal-pardot-form.html.twig

```diff
@@ -0,0 +1,58 @@
+{#
```

Comment:
This is a module-supplied template which could be overr-ridden at the theme layer.